### ISSUE TITLE
About train bug

### ISSUE THREAD

**Issue by AIwuya** [zhangpur/SR-LSTM#3]
when I use the train.py, have a bug like this
![image](https://user-images.githubusercontent.com/26964384/84516732-c8482d00-ad00-11ea-9344-120bb251ed9f.png)

In Processor.py line 136, train_loss, _ =self.train_epoch(epoch), can not use the loss.backward()
